astropy/io/votable/xmlutil.py

```diff
@@ -60,9 +60,7 @@ def check_token(token, attr_name, config=None, pos=None):
 
     As defined by XML Schema Part 2.
     """
-    if token is not None and not xml_check.check_token(token):
-        return False
-    return True
+    return token is None or xml_check.check_token(token)
```

Comment:
I guess it is just this change, right, for this PR? This certainly is an improvement.

Reply:
yes that's the only change here. It's only conveniently the last commit.